source/_components/sensor.mqtt.markdown

```diff
@@ -78,10 +78,10 @@ payload_not_available:
   required: false
   type: string
   default: offline
-json_attributes:
```

Comment:
`json_attributes` has not been removed?

Reply:
`json_attributes` is deprecated and will generate a deprecation warning when used.
Are there guidelines for how to document deprecated configuration variables?

@balloob, it's very related to the discussion in home-assistant/home-assistant#18726 I think, maybe you can give some guidance?

Reply:
let's keep it in the docs but mark it as deprecated.